docs/netlogoHPC/README.md

```diff
@@ -0,0 +1,79 @@
+## Introduction
+This document describes how to run a single Netlogo model on HPC using Apptainer container. We use the [Netlogo installation file](https://ccl.northwestern.edu/netlogo/6.3.0/NetLogo-6.3.0-64.tgz) as a base for our Apptainer container. The Apptainer container is then used to run the model in headless mode on HPC. The output is saved in a csv file.
+
+## Requirements
+- Apptainer (https://apptainer.org/)
+
+## Purpose
+This document describes how to run a single Netlogo model on HPC using Apptainer container. We use the [Netlogo installation file](https://ccl.northwestern.edu/netlogo/6.3.0/NetLogo-6.3.0-64.tgz) as a base for our Apptainer container. The Apptainer container is then used to run the model in headless mode on HPC. The output is saved in a csv file.
+
+## Build a container
+Build a singularity container from the singularity definition file. A Linux system with sudo rights is needed.
+
+```shell
+sudo apptainer build netlogo-headless.sif netlogo-headless.def
+```
+
+This will create an image `netlogo-headless.sif` from the corresponding definition file in the current directory. This image can be used to run the model in headless mode on HPC.
+
+## Prepare `job.sh` file
+
+This line of code will execute a `netlogo-headless.sh` command within a singularity container. `--model` is an argument that specifies the NetLogo model to be run. The `--experiment` argument is optional and can be used to specify the name of the experiment. The `--table` argument is optional and can be used to specify the name of the output file.
```

Comment:
may be better to show an example of normal usage and then show how you would use it with apptainer